packages/contracts/contracts/L1/rollup/CanonicalTransactionChain.sol

```diff
@@ -55,18 +61,76 @@ contract CanonicalTransactionChain is ICanonicalTransactionChain, Lib_AddressRes
     constructor(
         address _libAddressManager,
         uint256 _maxTransactionGasLimit,
-        uint256 _l2GasDiscountDivisor,
-        uint256 _enqueueGasCost
+        uint256 _enqueueGasCost,
+        uint256 _l2GasDiscountDivisor
```

Comment:
I reordered these. 
I forget why, but I tried to make it consistent throughout.